example/20-componentconfig-gardenlet.yaml

```diff
@@ -52,6 +52,7 @@ controllers:
     staleExtensionHealthChecks:
       enabled: true
     # threshold: 5m
+    managedResourceProgressingThreshold: 1h
```

Comment:
The gardenlet chart has to be adapted as well to support the new field in the GardenletConfiguration:

https://github.com/gardener/gardener/blob/9fae12b4cc29928cefd5f0cade44290e33378844/charts/gardener/gardenlet/templates/_helpers.tpl#L191-L205